<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1801</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #EAE7dc;</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="gameNameInputPage">
    <property name="minimumSize">
     <size>
      <width>1100</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1100</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QLabel" name="titleLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1071</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Games to Your List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="gameNameInputButton2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>781</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toGameInfoInputPageButton2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>781</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>1061</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 10pt &quot;Cambria&quot;;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gameNameLabel2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>250</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Game Title:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gameNameInputBox2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameInfoInputPage">
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QLabel" name="gameNameLabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="genreDescriptionLabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>1171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="gameInfoInputButton3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>810</y>
       <width>881</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLabel" name="popularGenreLabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>1171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="releaseDateLabel3">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>161</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Release Date: </string>
     </property>
    </widget>
    <widget class="QDateEdit" name="releaseDateInput3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>160</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>10</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1952</year>
       <month>10</month>
       <day>18</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="priceLabel3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Price:                 $</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="priceInput3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="maximum">
      <double>300.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="genreLabel3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Genre:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="genreInput3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>360</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <item>
      <property name="text">
       <string>Sandbox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real-Time Strategy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shooter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MOBA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Role-Playing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Simulation &amp; Sports</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Puzzle &amp; Party Games</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Action-Adventure</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Survival &amp; Horror</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Platformer</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tableDisplayPage">
    <widget class="QTableWidget" name="gameTableWidget4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>1751</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #ffffff;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Genre</string>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Release Date</string>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="titleLabel4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1741</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Your List of Games</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="sortButton4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>621</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortSelection4">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>270</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <item>
      <property name="text">
       <string>Genre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Price</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="rowDeleteButton4">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>170</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Delete Row</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowIndexBox4">
     <property name="geometry">
      <rect>
       <x>1550</x>
       <y>170</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="addGameButton4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>621</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Add Row</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="startPage">
    <widget class="QLabel" name="titleLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gaming Playlist Creator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>1161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>By: Jacoby King</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="toGameNameInputPageButton1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>810</y>
       <width>861</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #d8c3a5;</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameDescriptionLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>1161</width>
       <height>451</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Welcome to my game playlist creator program! This was built as
part of my final project for CIS 152, data structures. This program
allows the user to input any number of video game titles before
having you input the individual data for each game like release date,
price, and genre. Once the user is done entering all of the games for
their list a new page is displayed that contains a QTableWidget to
present all of the previously input data in a clear way. This table can
then be added onto, deleted from, or just sorted by data. Thank you
for checking out my program, I hope you enjoy.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
